flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalogConfig.java

```diff
@@ -32,4 +32,9 @@
 
 	// Partition related configs
 	public static final String PARTITION_LOCATION = "partition.location";
+
+	// config prefix for the table schema of a generic table
+	public static final String GENERIC_TABLE_SCHEMA_PREFIX = "generic.table.schema";
```

Comment:
consistent with `CatalogConfig.FLINK_PROPERTY_PREFIX`?

Reply:
`CatalogConfig.FLINK_PROPERTY_PREFIX` will be added to all the Flink properties. So we'll have sth like `flink.generic.table.schema.0...` and `flink.generic.table.schema.1...` in the final table properties.